src/components/facts/list/container.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component, PropTypes} from 'react'
+import {connect} from 'react-redux'
+
+import * as organizationActions from 'gModules/organizations/actions'
+
+import {FactList} from './list.js'
+
+function mapStateToProps(state) {
+  return {
+    organizations: state.organizations,
+    organizationFacts: state.facts.organizationFacts,
+  }
+}
+
+@connect(mapStateToProps)
+export class FactListContainer extends Component{
```

Comment:
This is needed to be called like this in the space, where it doesn't have immediate access to the organization.  It's a short query.